packages/admin-api-schema/lib/index.js

```diff
@@ -0,0 +1,85 @@
+const jsonSchema = require('./utils/json-schema');
+
+const getJSONDefinition = (version, name) => {
+    const definitionPath = `./${version}/${name}`;
+
+    try {
+        return require(definitionPath);
+    } catch (err) {
+        if (err.code === 'MODULE_NOT_FOUND') {
+            return undefined;
+        }
+    }
+};
+
+const get = ({version, definitionName}) => getJSONDefinition(version, definitionName);
```

Comment:
What is the purpose of this wrapper?

Reply:
It was meant to be a method that is exposed through the API and only would return values for definitions which are available in "list()" method. Changed ti's implementation so it returns `null` when invalid definition name is passed in :+1: